challenges/Backend Challenge/app.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask
+import requests
+
+app = Flask(__name__)
+arr = []
+
+@app.route('/')
+def index():
+	return "Hello World!"
+
+@app.route('/data', methods = ['GET'])
+def sortnumbers():
+	if len(arr) == 0:
+		return "Number list is empty"
+	arr.sort()
+	res = ""
+	for ele in arr:
+		res += str(ele) + " "
+	return res
```

Comment:
The description of the challenge asked for JSON formatting of the response. Is this response String JSON formatted?  If not could you complete the PR with code to JSON format the response?

Reply:
Please find the changes in app1.py. Thanks.